helm-chart/sefaria-project/templates/_helpers.tpl

```diff
@@ -175,3 +175,13 @@ preferredDuringSchedulingIgnoredDuringExecution:
             - "true"
 {{- end }}
 {{- end }}
+
+{{/*
+Setup complete tasks queue info
+*/}}
+{{- define "sefaria.tasks.internalQueues" }}
```

Comment:
@BrendanGalloway can you explain what this code is doing?

Reply:
The queues.tasks value should be derived from the environment to avoid collisions.  Ideally a user shouldn't be required to make sure the .tasks.queues.tasks value and the .deployEnv value align, so this sets up an internal variable with pre-build queue info, which is then merged with the value provided by the user before being passed to the app.  See the configmap CELERY_QUEUES value and the tasks pod args for usage